Makefile

```diff
@@ -1,14 +1,28 @@
-.PHONY: all doc doc-man doc-latex docclean
+.PHONY: all doc docman doclatex docclean help
 
-doc:
+# List all platforms
+pfs := $(foreach dir,$(wildcard platforms/*/Makefile),$(subst Makefile, , $(dir)))
+
+all: 		## Build all platforms
+	$(foreach pf, $(pfs), make -j$$(nproc) -C $(pf);)
```

Comment:
Other documentation related target below uses "$(MAKE)" variable instead of direct call to make.

Reply:
Ah yes good point !